src/main/resources/db/changelog/changes/13-create-shopping_cart-table.yaml

```diff
@@ -0,0 +1,27 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-shopping_cart-table
+      author: shastkiv
+      changes:
+        - createTable:
+            tableName: shopping_carts
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  constraints:
+                    primaryKey: true
+                    nullable: false
```

Comment:
primaryKey: true already means that nullable: false, so no need to add this explicitly

Reply:
Reason: liquibase.exception.DatabaseException: Column "USER_ID" must not be nullable